notebooks/tests/MsftRecommendations.ipynb

```diff
@@ -0,0 +1,51 @@
+{
+                 "cells": [
+                  {
+                   "cell_type": "code",
+                   "execution_count": null,
+                   "metadata": {
+                    "collapsed": true
+                   },
+                   "outputs": [],
+                   "source": [
+                    "# Use the Azure Machine Learning data collector to log various metrics\r\n",
```

Comment:
Could you please remove the data collection code from this notebook? It's intended to work on all Spark clusters.